website/cue/reference/components/sources/http.cue

```diff
@@ -128,6 +128,15 @@ components: sources: http: {
 				examples: ["vector_http_path"]
 			}
 		}
+		http_method: {
+			common:      false
+			description: "Specifies the action of the HTTP request."
+			required:    false
+			type: string: {
+				default: "POST"
+				examples: ["HEAD", "GET", "PUT", "POST", "PATCH", "DELETE"]
+			}
+		}
```

Comment:
I think this should be an `enum` to show users the only valid methods. See

https://github.com/vectordotdev/vector/blob/97e39a9ac1f61f02c61e549009ac1fe2c9ff6d17/website/cue/reference/components/sources/aws_s3.cue#L59-L72

For an example.